docs/operator-guide/customize_deployment.md

```diff
@@ -0,0 +1,46 @@
+# Customize Deployment
+
+Sometimes additional steps are required during application deployment. With the customization of the KubeRocketCI deployment, it's possible to override the deployment logic, which can be useful, for example, if you need to spin up and configure a database before deploying the main application, remove old information from the environment, redeploy the application, and initiate post-deployment configuration, tests, and so on.
+This is where custom deployment comes into play. This page provides comprehensive guidelines on how to adjust the deployment logic to cater your needs.
+
+## Deploy Custom Pipeline
+
+To customize your deployment pipeline, follow the steps below:
+
+1. Create the `TriggerTemplate` custom resource by adding the following label: 
+  ```
+  labels:
+      app.edp.epam.com/pipelinetype: deploy
+  ```
+
+!!! note
+    Please refer to the `TriggerTemplate` [example](https://github.com/epam/edp-tekton/blob/master/charts/pipelines-library/templates/triggers/cd/deploy.yaml) for more details. Remember to set your pipeline name in the spec.resourcetemplates.spec.pipelineRef.name parameter (line #33).
+
+2. Create the custom pipeline with your custom logic. Refer to the custom pipeline [example](https://github.com/epam/edp-tekton/blob/master/charts/pipelines-library/templates/pipelines/cd/deploy.yaml) for more details.
+
+3. Apply the created manifest files in the `edp` namespace.
+
+4. In the `Create stage` window of the KubeRocketCI portal, select the added trigger template in the corresponding window:
+
+  !![Select trigger template](../assets/operator-guide/select_trigger_template.png "Select trigger template")
+
+## Deploy Custom Pipeline to Remote Cluster (Optional)
```

Comment:
Why do we have this separate section? I see that here you just need to remember that if you remote the cluster, then don't forget the secret and nothing more

Reply:
Done